dataland-framework-toolbox/inputs/gdv/dataDictionary-GDV-VOEB-GDV-VÖB ESG questionnaire.csv

```diff
@@ -1,13 +1,13 @@
 Field Identifier,Category,Sub-Category,Field Name,Tooltip,Component,Options,Unit,Document-Support,Dependency,Show when value is,Mandatory Field
-0,General,Master Data,Berichtspflicht und Einwilligung zur Veröffentlichung,Ist das Unternehmen berichtspflichtig und mit der Veröffentlichung des Datensatzes auf Dataland einverstanden? Andernfalls ist eine weitere Dateneingabe nicht möglich.,Yes/No,,,,,,Yes
+0,General,Master Data,Berichtspflicht und Einwilligung zur Veröffentlichung,Ist das Unternehmen berichtspflichtig im Rahmen der CSRD-Richtlinie bzw. ist es gemäß den Offenlegungspflichten der Artikel 19a und 29a der Richtilinie 2023/34/EU zur Nachhaltigkeitsberichtserstattung verpflichtet? Ist das Unternehmen außerdem mit einer Veröffentlichung des Datensatzes auf Dataland einverstanden? Anderenfalls ist eine Dateneineingabe nicht möglich.,Yes/No,,,,,,Yes
 1,General,Master Data,(Gültigkeits) Datum,Datum bis wann die Information gültig ist,Date,,,,,,
-2,Allgemein,ESG Ziele,Existenz von ESG-Zielen,"Hat das Unternehmen spezifische ESG-Ziele/Engagements? Werden bspw. spezifische Ziele / Maßnahmen ergriffen, um das 1,5 Grad Ziel zu erreichen?",Yes/No,,,,0,Yes,
-2.1,Allgemein,ESG Ziele,Beschreibung der ESG-Ziele,Bitte geben Sie eine genaue Beschreibung der ESG-Ziele.,String,,,,2,Yes,
-2.2,Allgemein,ESG Ziele,Investitionen in Zielerreichung,Bitte geben Sie an wieviele Budgets/Vollzeitäquivalente für das Erreichen der ESG-Ziele zugewiesen wurden.,String,,,,2,Yes,
+2,Allgemein,ESG Ziele,Existenz von ESG-Zielen,"Hat das Unternehmen spezifische ESG-Ziele / Engagements? Werden bspw. spezifische Ziele / Maßnahmen ergriffen, um das 1,5 Grad Ziel zu erreichen?",Yes/No,,,,0,Yes,
+2.1,Allgemein,ESG Ziele,Beschreibung der ESG-Ziele,Bitte geben Sie eine genaue Beschreibung der ESG-Ziele.,FreeText,,,,2,Yes,
+2.2,Allgemein,ESG Ziele,Investitionen in Zielerreichung,Bitte geben Sie an wieviele Budgets/Vollzeitäquivalente für das Erreichen der ESG-Ziele zugewiesen wurden.,FreeText,,,,2,Yes,
 3,Allgemein,Sektoren,Sektoren mit hohen Klimaauswirkungen,Kann das Unternehmen einem oder mehreren Sektoren mit hohen Klimaauswirkungen zugeordnet werden?,Yes/No,,,,0,Yes,
-3.1,Allgemein,Sektoren,Auflistung der Sektoren,"Bitte geben Sie an, zu welchen Sektoren (mit hohen Klimaauswirkungen) das Unternehmen zugeordnet werden kann.",Multi-Select Dropdown,A; B; C; D; E; F; G; H; L,,,3,Yes,
+3.1,Allgemein,Sektoren,Auflistung der Sektoren,"Bitte geben Sie an, zu welchen Sektoren (mit hohen Klimaauswirkungen) das Unternehmen zugeordnet werden kann.",Multi-Select Dropdown,"A - Landwirtschaft, Forstwirtschaft und Fischerei| B - Bergbau und Gewinnung von Steinen und Erden|C - Verarbeitendes Gewerbe / Herstellung von Waren|D - Energieversorgung|E - Wasserversorgung; Abwasser & Abfallentsorgung; Beseitigungen von Umweltverschmutzungen|F - Baugewerbe/Bau|G - Handel; Instandhaltung und Reparatur von Kraftfahrzeugen|H - Verkehr und Lagerhaltung| L - Grundstücks- und Wohnungswesen",,,3,Yes,
```

Comment:
There are spaces before some of the elements.

Reply:
That should not do a difference, because the function `trim()` is being used in the devtools on each option, and it removes spaces at the beginning and end of the string